client/a8c-for-agencies/components/a4a-migration-offer/index.tsx

```diff
@@ -30,13 +30,12 @@ const MigrationOfferBody = () => {
 		<>
 			<p className="a4a-migration-offer__description">{ description }</p>
 			<Button
-				className="a4a-migration-offer__chat-button"
```

Comment:
I searched through the codebase and I couldn't find where this class is defined, so I assume that is why it is being removed.